include/ofxOld.h

```diff
@@ -0,0 +1,99 @@
+/*
+Software License :
+
+Copyright (c) 2014-2015, The Open Effects Association Ltd. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimer in the documentation
+      and/or other materials provided with the distribution.
+    * Neither the name The Open Effects Association Ltd, nor the names of its 
+      contributors may be used to endorse or promote products derived from this
+      software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+*/
+
+#ifndef _ofxOLD_h_
+#define _ofxOLD_h_
+
+ 
+/** @brief String to label images with YUVA components
+--ofxImageEffects.h
+Note, this has been deprecated in v1.3
+removed in v1.4
+ */
+#define kOfxImageComponentYUVA "OfxImageComponentYUVA"
+
+/** @brief Indicates whether an effect is performing an analysis pass.
+--ofxImageEffects.h
+   - Type - int X 1
+   - Property Set -  plugin instance (read/write)
+   - Default - to 0
+   - Valid Values - This must be one of 0 or 1
+
+This feature has been deprecated - officially commented out v1.4.
```

Comment:
No.  That was only when trying to keep the #defines inline -- that's what prompted me to ask for alternative ideas.  I think all you need is the @deprecated. (Plus switch the 1.3 to 1.4, but that should be a separate commit and PR, I can do that if desired.)